src/index.js

```diff
@@ -28,15 +29,32 @@ module.exports = {
     start: compileProject,
     watcher: async function ({ filename, /* event, */ inventory }) {
       if (filename.endsWith('.ts') || filename.endsWith('.tsx')) {
-        // Second pass filter by Lambda dir
+        let { cwd } = inventory.inv._project
+        let globalTsConfig = getTsConfig(cwd)
+        let tsCompilerOptions = require(globalTsConfig).compilerOptions
```

Comment:
We need to `fs/promises` `readFile` this; requiring the JSON will load it into memory, and changes by the user will not be picked up by the plugin over time.